conformance/lang/expressions/literals/floating_point_literal.balt

```diff
@@ -0,0 +1,6202 @@
+
+    io:println(printFloatingPoint(25F)); // @output 25.0
+
+    io:println(printFloatingPoint(25d)); // @output 25
+
+    io:println(printFloatingPoint(25D)); // @output 25
+}
+
+function printFloatingPoint(float|decimal x) returns float|decimal {
+    return x;
+}
+
+Test-Case: output
+Description: Test DecimalFloatingPointNumber with return statement.
+Labels: FloatingPointTypeSuffix, return-stmt, union-type
+
+function init() {
+    io:println(printDecimalFloatingPoint1()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint2()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint3()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint4()); // @output 2.5E+175
+
+    io:println(printDecimalFloatingPoint5()); // @output 2.5E-173
+
+    io:println(printDecimalFloatingPoint6()); // @output 2.5E+175
+
+    io:println(printDecimalFloatingPoint7()); // @output 25.0
+
+    io:println(printDecimalFloatingPoint8()); // @output 25
+
+    io:println(printDecimalFloatingPoint9()); // @output 25.0
+
+    io:println(printDecimalFloatingPoint21()); // @output 245.32
+
+    io:println(printDecimalFloatingPoint10()); // @output 0.99
+
+    io:println(printDecimalFloatingPoint11()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint12()); // @output 2.4332E176
+
+    io:println(printDecimalFloatingPoint13()); // @output 9.9E173
+
+    io:println(printDecimalFloatingPoint14()); // @output 2.5E-173
+
+    io:println(printDecimalFloatingPoint16()); // @output 2.4332E176
+
+    io:println(printDecimalFloatingPoint15()); // @output 9.9E-175
+
+    io:println(printDecimalFloatingPoint17()); // @output 25.0
+
+    io:println(printDecimalFloatingPoint18()); // @output 25.0
+
+    io:println(printDecimalFloatingPoint19()); // @output 245.32
+
+    io:println(printDecimalFloatingPoint20()); // @output 245.32
+
+    io:println(printDecimalFloatingPoint23()); // @output 0.99
+
+    io:println(printDecimalFloatingPoint22()); // @output 0.99
+
+    io:println(printDecimalFloatingPoint24()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint25()); // @output 2.50E+175
+
+    io:println(printDecimalFloatingPoint27()); // @output 2.4532E176
+
+    io:println(printDecimalFloatingPoint26()); // @output 2.4532E+176
+
+    io:println(printDecimalFloatingPoint29()); // @output 9.9E173
+
+    io:println(printDecimalFloatingPoint28()); // @output 9.9E+173
+
+    io:println(printDecimalFloatingPoint30()); // @output 2.5E175
+
+    io:println(printDecimalFloatingPoint31()); // @output 2.50E-173
+
+    io:println(printDecimalFloatingPoint33()); // @output 2.4532E+176
+
+    io:println(printDecimalFloatingPoint32()); // @output 2.4532E176
+
+    io:println(printDecimalFloatingPoint34()); // @output 9.9E+173
+
+    io:println(printDecimalFloatingPoint35()); // @output 9.9E-175
+
+    io:println(printDecimalFloatingPoint36()); // @output 25.0
+
+}
+
+function printDecimalFloatingPoint1() returns float|decimal {
```

Comment:
We don't have to change here, but taking this as an example for future tests, we can make tests simpler by avoiding testing the same thing over and over again.

We've already tested different literals against float|decimal, so we don't have to test in every context. Just one or two would be enough.